### PR TITLE
new icon: polygon (plain, original)

### PR DESCRIPTION
**Pull request by EthanDevelops**
# **New icon section**

## Double check these details before you open a PR

- [x] PR does not match another non-stale PR currently opened
- [x] PR name matches the format *new icon: <i>Icon name</i> (<i>versions separated by comma</i>)*. More details [here](https://github.com/devicons/devicon/blob/develop/CONTRIBUTING.md#overview)
- [x] The PR's base is the `develop` branch.
- [X] Your icons are inside a folder as seen [here](https://github.com/devicons/devicon/blob/develop/CONTRIBUTING.md#organizational-guidelines)
- [x] SVG matches the standards laid out [here](https://github.com/devicons/devicon/blob/develop/CONTRIBUTING.md#svgStandards)
- [x] A new object is added in the `devicon.json` file as seen [here](https://github.com/devicons/devicon/blob/develop/CONTRIBUTING.md#-updating-the-deviconjson-)

## Link to prove your SVG is correct and up-to-date

[]<https://polygon.technology>

### CODE REVIEW

**Comment by EthanDevelops** [comment]
Also fixed the devicon.json and resolved conflicts with my branch.

File: icons/discordjs/discordjs-original-wordmark.svg [hunk 1]
```diff
@@ -13,4 +13,4 @@
...
   </g>
  </g>
-</svg>
+</svg>
```
Comment (Panquesito7) [inline icons/discordjs/discordjs-original-wordmark.svg:16]: Missing an endline.
```suggestion
</svg>


```

File: devicon.json [hunk 2]
```diff
@@ -3315,6 +3314,47 @@
...
+            "svg": [
+                "original",
+                "plain"
+            ],
+            "font": [
+                "plain"
+            ]
+        },
+        "color": "#38285B",
+        "aliases": [ "matic" ]
```
Comment (Panquesito7) [inline devicon.json:3356]:
```suggestion
        "aliases": []
```

**Comment by Thomas-Boi** [comment]
Hi Ethan,

While the icon version looks great, it seems like your SVG contains more than 1 color. Usually, Icomoon would not render this properly. However, it seems to work in this case which is the first time this had happened. Icomoon automatically cut out the path from the circle for us (which is usually required to be done manually).
image

I'm leaning towards accepting the PR but all the other `plain` SVG versions have only 1 color in them. It might be prudent to keep the standard and modify the SVG to have only 1 color. 

@amacado @Panquesito7 what do you think? Should we accept the PR as is?

File: icons/discordjs/discordjs-original-wordmark.svg [hunk 3]
```diff
@@ -14,3 +14,4 @@
   </g>
  </g>
 </svg>
+
```
Comment (Panquesito7) [inline icons/discordjs/discordjs-original-wordmark.svg:17]:
```suggestion
```

**Review by Panquesito7 — COMMENTED** [review]
> @amacado @Panquesito7 what do you think? Should we accept the PR as is?

I think we should update properly, then merge. 🙂

**Comment by EthanDevelops** [comment]
@Thomas-Boi I believe I fixed it

**Review by Panquesito7 — APPROVED** [review]
Awesome work! 🚀 Thanks so much for your patience and contribution! 😄👍

**Review by Thomas-Boi — APPROVED** [review]
Sweet, thank you @EthanDevelops for fixing the SVG!

**Pull request merged**